integration-tests/container/README.md

```diff
@@ -23,6 +23,18 @@ The intention of this version number is to ensure consistency across master
 testing and release testing, whereby releases are pegged to a specific QA image
 version.
 
+To build all images run
+
+`make build-all`
```

Comment:
I think these targets are now `build` and `build-and-push` with the above makefile changes